### ISSUE TITLE
Can't send Email without authentification

### ISSUE THREAD

**Issue by katyucha** [gravitee-io/issues#1692]
I try to send email without server authentification : Can't 

Here's my configuration 
```
email:
  enabled: true
  host: 192.168.1.25
  subject: "[Gravitee.io] %s"
  port: 25
  from: admin@keycloak.at.work
```
(I try with properties: auth : false  too  ) 

I sign up a new account or I try to send another password from the users settings : 

Always same error : 

```
09:19:00.363 [SimpleAsyncTaskExecutor-2] ERROR o.s.a.i.SimpleAsyncUncaughtExceptionHandler - Unexpected error occurred invoking async method 'public void io.gravitee.management.service.impl.
EmailServiceImpl.sendAsyncEmailNotification(io.gravitee.management.service.EmailNotification)'.                                                                                               
io.gravitee.management.service.exceptions.TechnicalManagementException: Error while sending email notification                                                                                
        at io.gravitee.management.service.impl.EmailServiceImpl.sendEmailNotification(EmailServiceImpl.java:106)                                                                              
        at io.gravitee.management.service.impl.EmailServiceImpl.sendAsyncEmailNotification(EmailServiceImpl.java:113)                                                                         
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)                                                                                                                        
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)                                                                                                      
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)                                                                                              
        at java.lang.reflect.Method.invoke(Method.java:498)                                                                                                                                   
        at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:343)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:197)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)
        at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:294)
        at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:98)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:185)
        at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:115)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.lang.Thread.run(Thread.java:748)
Caused by: org.springframework.mail.MailAuthenticationException: Authentication failed; nested exception is javax.mail.AuthenticationFailedException: No authentication mechansims supported $y both server and client
        at org.springframework.mail.javamail.JavaMailSenderImpl.doSend(JavaMailSenderImpl.java:438)
        at org.springframework.mail.javamail.JavaMailSenderImpl.send(JavaMailSenderImpl.java:359)
        at org.springframework.mail.javamail.JavaMailSenderImpl.send(JavaMailSenderImpl.java:354)
        at io.gravitee.management.service.impl.EmailServiceImpl.sendEmailNotification(EmailServiceImpl.java:103)
        ... 14 common frames omitted
Caused by: javax.mail.AuthenticationFailedException: No authentication mechansims supported by both server and client
        at com.sun.mail.smtp.SMTPTransport.authenticate(SMTPTransport.java:765)
        at com.sun.mail.smtp.SMTPTransport.protocolConnect(SMTPTransport.java:685)
        at javax.mail.Service.connect(Service.java:295)
        at org.springframework.mail.javamail.JavaMailSenderImpl.connectTransport(JavaMailSenderImpl.java:515)
        at org.springframework.mail.javamail.JavaMailSenderImpl.doSend(JavaMailSenderImpl.java:435)
        ... 17 common frames omitted

``` 

When I use linux command like 'mail' in anonymous , no problem to send email. 

### Your Environment

* Version used: 4.5.0
* Browser Name and version: Firefox
* Operating System and version: Linux Debian 9

**Comment by brasseld** [comment]
Please tell us which version of gravitee.io you're using.

**Comment by brasseld** [comment]
I think it has been fixed in 1.20.4 according to this issue https://github.com/gravitee-io/issues/issues/1652

Please provide your feedback.

Regards,

**Comment by katyucha** [comment]
upgraded
solved
sorry !
